lua/lspconfig/configs.lua

```diff
@@ -86,8 +86,9 @@ function configs.__newindex(t, config_name, config_def)
         api.nvim_command(
           string.format(
             "autocmd %s unsilent lua require'lspconfig'[%q].manager.try_add_wrapper()",
-            'BufReadPost ' .. root_dir .. '/*',
-            config.name
+            'BufReadPost %s %s/*',
+            config.name,
+            util.path.sanitize_root_dir(root_dir, true)
```

Comment:
```suggestion
            util.path.sanitize_path(root_dir, true)
```
I don't think `sanitize_root_dir` exists